CONTRIBUTE.adoc

```diff
@@ -98,7 +98,7 @@ Examples:: These are examples
 Even more examples...
 ====
 
-== Publishing
+== Publishing as HTML
```

Comment:
It's really publishing, meaning to our website at https://redhat-cop.github.io/automation-good-practices/. Please rename more aptly to "Publish for the website".

Reply:
done via 9be882a
feel free to drop 29581f4 I added that one as the link to the website was not immediately obvious to me when lookuing at the source repo.